README.md

```diff
@@ -170,6 +170,8 @@ You can enable the web search through an API by adding `YDC_API_KEY` ([docs.you.
 
 You can also simply enable the local google websearch by setting `USE_LOCAL_WEBSEARCH=true` in your `.env.local` or specify a SearXNG instance by adding the query URL to `SEARXNG_QUERY_URL`.
 
```

Comment:
in the websearch readme, should we add somewhere that `npx playwright install` is needed to run the websearch?

Reply:
That shouldn't be needed anymore since I added `@playwright/browser-chromium` to the `package.json` but I can't test that it works on my system